includes/Modules/Site_Verification.php

```diff
@@ -96,6 +102,14 @@ function () {
 			}
 		);
 
+		$clear_verification_meta_cache = function ( $meta_id, $object_id, $meta_key ) {
+			if ( $this->user_options->get_meta_key( Verification_Meta::OPTION ) === $meta_key ) {
+				delete_transient( self::TRANSIENT_VERIFICATION_META_TAGS );
```

Comment:
Oh I just noticed again that we need to access transients here too. Since it's so specific though, maybe we should still instantiate `Transients` as a once-off object here _and_ further down in the other method?